docs/en/_snippets/_add_superset_detail.md.bak

```diff
@@ -0,0 +1,47 @@
+<details>
```

Comment:
@gingerwizard what are these .bak files? Do they need to be commited?

Reply:
should add *.md.bak to .gitignore